### PR TITLE
fix-security-impact

### PR DESCRIPTION
**Pull request by Ximinhan**
when adjust security impact only adjust to higher impact

### CODE REVIEW

File: elliottlib/cli/attach_cve_flaws_cli.py [hunk 1]
```diff
@@ -215,9 +215,12 @@ def get_updated_advisory_rhsa(logger, cve_boilerplate: dict, advisory: Erratum,
 
     highest_impact = get_highest_security_impact(flaw_bugs)
     if highest_impact != advisory.security_impact:
-        logger.info(f'Adjusting advisory security impact from {advisory.security_impact} to {highest_impact}')
-        advisory.update(security_impact=highest_impact)
-        updated = True
+        if compare_security_impact(advisory.security_impact, highest_impact):
```
Comment (thegreyd) [inline elliottlib/cli/attach_cve_flaws_cli.py:218]: Why not have 
`if impact[advisory.security_impact] < impact[highest_impact]` in this line?

File: elliottlib/bzutil.py [hunk 2]
```diff
@@ -1016,6 +1016,11 @@ def get_highest_security_impact(bugs):
     return 'Low'
 
 
+def compare_security_impact(current_impact, target_impact):
+    impact = {'Critical': 4, 'Important': 3, 'Moderate': 2, 'Low': 1}
```
Comment (thegreyd) [inline elliottlib/bzutil.py:1020]: We can move this to constants.py

**Comment by thegreyd** [comment]
Looks good with minor comments

File: elliottlib/cli/attach_cve_flaws_cli.py (hunk 1 above)
Comment (Ximinhan) [reply elliottlib/cli/attach_cve_flaws_cli.py]: good point, updated

**Pull request merged**